operations/devices.md

```diff
@@ -45,13 +46,14 @@ Returns all devices in the enterprise.
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the device. |
+| `Type` | [Device type](#device-type) | required | Type of the device. |
 | `Name` | string | required | Name of the device. |
-| `Type` | string | required | [Type](#device-type) of the device. |
+| `Identifier` | string | optional | Device identifier (for internal purposes). |
 
-### Device type
+#### Device type
 
-* `Printer` - [Printer command data](commands.md#printer-command-data)
-* `PaymentTerminal` - [Payment terminal command data](commands.md#payment-terminal-command-data)
-* `PassportScanner` - [Passport scanner command data](commands.md#passport-scanner-command-data)
-* `FiscalMachine` - [Fiscal machine command data](commands.md#fiscal-machine-command-data)
-* `KeyCutter` - [Key cutter command data](commands.md#key-cutter-command-data)
+* `Printer` - [Printer command data](https://mews-systems.gitbook.io/connector-api/operations/commands#printer-command-data)
```

Comment:
@jnv I notice the links here are fully resolved links, rather than relative links. Just flagging that up, not sure if it might be a problem going forward, on the surface it does make the docs less portable because it ties it into the rendering tool and its URL. What if we changed from **GitBook**, or used a Mews domain like `api.mews.com`?

Reply:
Follow up: https://mews.atlassian.net/browse/CON-2322